test_newsletter_subscription.py

```
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
```

Ruff (0.14.7)

Shebang is present but file is not executable

(EXE001)